spec.html

```diff
@@ -38,6 +38,6 @@ <h1>CreateDataPropertyOnObject Functions</h1>
     1. Assert: Type(_O_) is Object.
     1. Assert: _O_ is an extensible ordinary object.
     1. Let _propertyKey_ be ? ToPropertyKey(_key_).
-    1. Perform ? CreateDataPropertyOrThrow(_O_, _propertyKey_, _value_).
+    1. Perform ! CreateDataPropertyOrThrow(_O_, _propertyKey_, _value_).
```

Comment:
If "!", does `CreateDataPropertyOrThrow` still differ from `CreateDataProperty`?

Reply:
Yes, that can return false - the combo of OrThrow and ! ensures it can never throw; otherwise we’d have to assert that the operation result was true.

Reply:
Thanks, makes sense, I misunderstood how they differed.

Reply:
To be clear, semantically they're equivalent, since `CreateDataProperty` cannot, in this case, ever actually return false.